README.md

```diff
@@ -1,24 +1,271 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Configuration
 
-* Database creation
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- PROJECT DESCRIPTION -->
 
-* ...
+# 📖 [Spend Right Budget App] <a name="about-project"></a>
+
+**Spend Right** is a full stack ruby on rails web application on a postgreSQL database that is built to manage your finances by grouping your expenses or transactions into categories and helping you visualize how much your are spending and where you are spending the most amount of money.
+
+## 🛠 Built With <a name="built-with">Ruby on Rails</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://render.com/">Render</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Spash screen to intoduce user to app**
+- **Total amount of transactions in each category**
+- **Ability to add new category**
+- **Ability to add new transaction to a category**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://spend-right.onrender.com)
+
```

Comment:
- [ ] In the live version. After pressing the sign-up button, I'm getting this error. Please fix this.

![Screenshot from 2023-02-25 04-35-01](https://user-images.githubusercontent.com/26767383/221319314-fd4256d2-7cde-442c-ab29-10c03a990838.png)

Reply:
Thank you for the feedback. I will check for the cause of this problem as soon as possible